_data/projects/VisitMeet.yml

```diff
@@ -0,0 +1,11 @@
+name: VisitMeet
+desc: VisitMeet is a cheapeast way to travel and develop 3rd world countries at the same time.
+site: https://github.com/VisitMeet/visitmeet
+tags:
+- oss
+- web
+- ruby on rails
+- javascript
+upforgrabs:
+  name: VisitMeet
+  link: https://github.com/VisitMeet/visitmeet/labels/up-for-grabs
```

Comment:
Will fix that ASAP :)
On Mar 28, 2016 2:48 AM, "Brendan Forster" <notifications@github.com> wrote:

> In _data/projects/VisitMeet.yml
> <https://github.com/up-for-grabs/up-for-grabs.net/pull/389#discussion_r57534792>
> :
>
> > @@ -0,0 +1,11 @@
> > +name: VisitMeet
> > +desc: VisitMeet is a cheapeast way to travel and develop 3rd world countries at the same time.
> > +site: https://github.com/VisitMeet/visitmeet
> > +tags:
> > +- oss
> > +- web
> > +- ruby on rails
> > +- javascript
> > +upforgrabs:
> > +  name: VisitMeet
> > +  link: https://github.com/VisitMeet/visitmeet/labels/up-for-grabs
>
> I can see you're using help wanted for some of the issues, instead of
> up-for-grab.
>
> In that case, your link here should be:
> https://github.com/VisitMeet/visitmeet/labels/help%20wanted
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/up-for-grabs/up-for-grabs.net/pull/389/files/7cdcd2fe987482ad2a2012e802998aa227d88c14#r57534792>
>